src/main/java/com/carrotcreative/recyclercore/adapter/RecyclerCoreModel.java

```diff
@@ -1,15 +1,5 @@
 package com.carrotcreative.recyclercore.adapter;
 
-import android.view.LayoutInflater;
-import android.view.ViewGroup;
-
-public abstract class RecyclerCoreModel {
-
-    public String getViewType()
-    {
-        return getClass().getCanonicalName();
-    }
-
-    public abstract RecyclerCoreController buildController(LayoutInflater inflater, ViewGroup parent);
-
+public abstract class RecyclerCoreModel
```

Comment:
Is there any reason to even mandate that it's a RecyclerCoreModel anymore?  What's stopping us from just using a POJO now?